src/components/SuggestedJourneys/SuggestedJourneys.module.scss

```diff
@@ -0,0 +1,93 @@
+@import '../../styles/base.scss';
+
+.journey {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    flex-grow: 1;
+
+    h2 {
+        margin: 0;
+    }
+
+    a {
+        //position: relative;
+        color: rgba(255, 255, 255, 1);
+        text-decoration: none;
+        background-color: rgb(243, 190, 74);
+        font-weight: 250;
+        font-size: 3em;
+        display: block;
+        padding: 4px;
+        -webkit-border-radius: 8px;
```

Comment:
It's fantastic that you're addressing cross-browser support here, but in truth it's mostly no longer necessary to write out these browser-specific properties (with very few exceptions). Most modern browsers support the un-prefixed version, and for the cases we find that don't there is a plugin in our build system that automatically applies prefixes when compiling the `.scss`. to `.css`.